actiontext/lib/action_text/attribute.rb

```diff
@@ -30,7 +30,11 @@ module Attribute
       #
       # * <tt>:encrypted</tt> - Pass true to encrypt the rich text attribute. The encryption will be non-deterministic. See
       #   +ActiveRecord::Encryption::EncryptableRecord.encrypts+. Default: false.
-      def has_rich_text(name, encrypted: false)
+      #
+      # * <tt>:strict_loading</tt> - Pass true to force strict loading. When
+      #   omitted, <tt>strict_loading:</tt> will be set to the value of the
+      #   <tt>strict_loading_by_default</tt> class attribute (false by default).
+      def has_rich_text(name, encrypted: false, strict_loading: strict_loading_by_default)
```

Comment:
Would it be better to default to `strict_loading: nil`, then omit the key from `has_one` if it isn't provided?

Reply:
I think so, though it should have the same outcome either way.